build/build.proj

```diff
@@ -12,6 +12,66 @@
     </CommonMSBuildProperties>
   </PropertyGroup>
 
+  <!-- The project that builds the VSIX -->
+  <PropertyGroup>
+    <VSIXProject>$(RepositoryRootDirectory)src\NuGet.Clients\NuGet.VisualStudio.Client\NuGet.VisualStudio.Client.csproj</VSIXProject>
+  </PropertyGroup>
+
+  <!-- Find all test projects  -->
+  <ItemGroup Condition=" '$(IsXPlat)' != 'true' ">
+    <CoreUnitTestProjects Include="$(RepositoryRootDirectory)test\NuGet.Core.Tests\*\*.csproj"
```

Comment:
docs.proj is using these properties. 
Can you ensure that the generation still works correctly?

Reply:
Fixed, and when I run it now, the only change is the timestamp written to the file.